<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VSCVWidget</class>
 <widget class="QWidget" name="VSCVWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb( 100,100,100);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb( 100,100,100);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(0, 0, 0)</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="labelName">
           <property name="styleSheet">
            <string notr="true">color : rgb(247, 247, 247);
font: 8pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="labelRecording">
           <property name="styleSheet">
            <string notr="true">color : red;
font: 8pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="VideoWidget" name="video" native="true">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(0, 0, 0)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="videoControl" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(42, 43, 44)</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="timelineZoom">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;Arial&quot;;</string>
           </property>
           <item>
            <property name="text">
             <string>1   min</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5   min</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>30 min</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2   hour</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1   day</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1   week</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2   week</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="VSCSimpleTl" name="widgetTimeline" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbPause">
           <property name="minimumSize">
            <size>
             <width>19</width>
             <height>19</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>19</width>
             <height>19</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../main/vscmainwindows.qrc">
             <normaloff>:/pb/resources/pause.png</normaloff>:/pb/resources/pause.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>17</width>
             <height>17</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbPlay">
           <property name="minimumSize">
            <size>
             <width>19</width>
             <height>19</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>19</width>
             <height>19</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../main/vscmainwindows.qrc">
             <normaloff>:/pb/resources/play.png</normaloff>:/pb/resources/play.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>17</width>
             <height>17</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbInstantpb">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>19</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>19</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(80, 80, 80);
color : rgb(247, 247, 247);
font: 75 6pt &quot;Arial&quot;;</string>
           </property>
           <property name="locale">
            <locale language="Colognian" country="Germany"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../main/vscmainwindows.qrc">
             <normaloff>:/pb/resources/instantpb.jpg</normaloff>:/pb/resources/instantpb.jpg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>19</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbRecord">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>19</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>19</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(80, 80, 80);
color : rgb(247, 247, 247);
font: 75 6pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../main/vscmainwindows.qrc">
             <normaloff>:/pb/resources/record.jpg</normaloff>:/pb/resources/record.jpg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>19</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
     <zorder>widget</zorder>
     <zorder>videoControl</zorder>
     <zorder>video</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QWidget</extends>
   <header>videowidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VSCSimpleTl</class>
   <extends>QWidget</extends>
   <header>vscsimpletl.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../main/vscmainwindows.qrc"/>
 </resources>
 <connections/>
</ui>
